types/react-redux/index.d.ts

```diff
@@ -7,9 +7,15 @@
 //                 Frank Tan <https://github.com/tansongyang>
 //                 Nicholas Boll <https://github.com/nicholasboll>
 //                 Dibyo Majumdar <https://github.com/mdibyo>
+//                 Prashant Deva <https://github.com/pdeva>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.4
-
+// There is a known issue in TypeScript, which doesnt allow decorators to change the signature of the classes 
```

Comment:
I'm not sure if this comment should be part of the rest of this "block" -- could you move it down to where the connect function is declared?

Reply:
its here so its obvious to the person using this type definition.
react-redux *is* eseentially just the connect() function. as soon as someone use this definition, they will be bombarded with a bunch of errors. so better to put this at the very top instead of hoping they will hunt and find this message.

Reply:
Ok, then can you at least separate it from the version comment by a newline? I'd like to see it as its own thing, rather than visually blending in.

Reply:
@pdeva not required for merge, but it would be nice to have some mention of this issue down where the connect function is declared as well. If someone is using ts-server or an IDE, they will be taken straight to the definition, not the top of the file. Something like "See file header for a known issue about using the connect function as a decorator." would work.